e2e/scaling/scaling.go

```diff
@@ -165,3 +167,80 @@ func (tc *ScalingE2ETest) TestScalingNamespaces(f *framework.F) {
 		"Nomad e2e testing", false, nil, &aWriteOpts)
 	f.NoError(err)
 }
+
+// TestScalingBasic performs basic scaling e2e tests within a single namespace using
+// using a SystemScheduler.
+func (tc *ScalingE2ETest) TestScalingBasicWithSystemSchedule(f *framework.F) {
+	t := f.T()
+	nomadClient := tc.Nomad()
+
+	// Register a system job with a scaling policy without a group count, it should
+	// default to 1.
+
+	jobID := "test-scaling-" + uuid.Generate()[0:8]
+	e2eutil.RegisterAndWaitForAllocs(t, nomadClient, "scaling/input/namespace_default_system.nomad", jobID, "")
+
+	jobs := nomadClient.Jobs()
+	allocs, _, err := jobs.Allocations(jobID, true, nil)
+	require.NoError(t, err)
```

Comment:
Do we use the `must` package in E2E tests as well?

Reply:
This package is really old that's why it uses `require`, we always use `must` now, but I didn't want to introduce one test that is different and changing all the tests in the package would make this PR a lot bigger.